funnel/models/event.py

```diff
@@ -50,6 +50,11 @@ class Event(BaseScopedNameMixin, db.Model):
     participants = db.relationship('Participant', secondary='attendee', backref='events', lazy='dynamic')
     __table_args__ = (db.UniqueConstraint('proposal_space_id', 'name'),)
 
+    @classmethod
+    def get_name(cls, space, title):
```

Comment:
At the risk of being verbose, I'd call this `get_name_from_title` so it's obvious what it does when you look at the call elsewhere in code.